.github/workflows/issue-autorespond-and-close.yml

```
@@ -0,0 +1,14 @@
+name: Auto respond and close issue
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  invoke_reusable_workflow:
```

Nitpick: This job name is too generic. I know it's the only one in the file (for now) but I think we should give it something more appropriate in case we ever add new jobs in the future.

```suggestion
  auto-respond-and-close-issue:
```

---

good point.